core/src/main/java/com/linecorp/armeria/common/AbstractHttpMessageBuilder.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+import static com.google.common.base.Preconditions.checkState;
+import static java.util.Objects.requireNonNull;
+
+import java.util.Locale;
+
+import org.reactivestreams.Publisher;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.errorprone.annotations.FormatMethod;
+import com.google.errorprone.annotations.FormatString;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.internal.common.JacksonUtil;
+
+abstract class AbstractHttpMessageBuilder {
+
+    @Nullable
+    private HttpData content;
+
+    @Nullable
+    private Publisher<? extends HttpData> publisher;
+
+    @Nullable
+    protected HttpData getContent() {
+        return content;
+    }
+
+    @Nullable
+    protected Publisher<? extends HttpData> getPublisher() {
+        return publisher;
+    }
```

Comment:
We usually don't use `get` prefix for getters.
```suggestion
    @Nullable
    protected final HttpData content() {
        return content;
    }

    @Nullable
    protected final Publisher<? extends HttpData> pblisher() {
        return publisher;
    }
```